src/panoptes/pocs/scheduler/observation/observation.py

```diff
@@ -1,16 +1,18 @@
 import os
 from collections import OrderedDict
-
 from astropy import units as u
+
+from panoptes.utils.utils import get_quantity_value
+
 from panoptes.pocs.base import PanBase
 from panoptes.pocs.scheduler.field import Field
 
 
 class Observation(PanBase):
```

Comment:
We should probably do a proper abstract base class, but for now can you rename the file here to `base.py` and then change them imports in files above? That will make it easier to adjust a bit in the future.

```suggestion
class ObservationBase(PanBase):
```

Reply:
I like the idea of changing to `base.py` but I think changing `Observation` to `ObservationBase` is unnecessary (already implied by the module name) and would mean all `Observation` objects currently in `POCS` would have to change to `ObservationBase` objects, which doesn't seem right.